packages/react-mutation-mapper/src/component/clinvar/ClinVarSummary.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+    DefaultTooltip,
+    pluralize,
+    TruncatedText,
+} from 'cbioportal-frontend-commons';
+import { getClinVarId } from 'cbioportal-utils';
+import { ClinVar, MyVariantInfo } from 'genome-nexus-ts-api-client';
+import _ from 'lodash';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+export type ClinVarSummaryProps = {
+    myVariantInfo?: MyVariantInfo;
+};
+
+export enum ClinVarOrigin {
+    GERMLINE = 'germline',
+    SOMATIC = 'somatic',
+}
+
+export type RcvData = {
+    origin: string;
+    evidences: {
+        clinicalSignificance: string;
+        count: number;
+    }[];
+};
+
+export type RcvCountMap = {
+    [origin: string]: { [clinicalSignificance: string]: number };
+};
+
+export function getClinVarInterpretation(rcvData: RcvData[]) {
+    return rcvData.map(d => (
+        <div key={d.origin}>
+            <strong>{`${_.upperFirst(d.origin)}: `}</strong>
+            {d.evidences
+                .map(
+                    e =>
+                        `${e.clinicalSignificance} (${e.count} ${pluralize(
+                            'evidence',
+                            e.count
+                        )})`
+                )
+                .join(', ')}
+        </div>
+    ));
+}
+
+export function getRcvCountMap(clinVar: ClinVar): RcvCountMap {
+    const filteredRcv = clinVar.rcv.filter(
+        rcv =>
+            rcv.origin === ClinVarOrigin.GERMLINE ||
+            rcv.origin === ClinVarOrigin.SOMATIC
+    );
+
+    // first map by origin, then count evidence number for each origin group
+    const rcvMap = _.groupBy(filteredRcv, d => d.origin);
+
+    return _.mapValues(rcvMap, rcvs =>
+        _.countBy(rcvs, rcv => rcv.clinicalSignificance)
+    );
+}
+
+export function getRcvData(rcvCountMap: RcvCountMap): RcvData[] {
+    return _.map(
```

Comment:
pretty sure _.map will already iterate over object.    (val, key)=>{}

Reply:
good point, I'll update this